cmd/tracee-ebpf/internal/flags/flags-filter.go

```diff
@@ -37,6 +37,8 @@ The field 'set' selects a set of events to trace according to predefined sets, w
 
 The special 'follow' expression declares that not only processes that match the criteria will be traced, but also their descendants.
 
+The field 'net' is required when tracing network events. without it, tracee wouldn't know which interface to sniff over
```

Comment:
The documentation here should specify what the `net` filter is, not just that it is a mandatory field for net events.
I recommend writing the following:
```suggestion
The field 'net' specifies which interfaces to monitor when tracing network events.
Notice that the 'net' field is mandatory when tracing network events.
````